pkg/controller/control.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/longhorn/longhorn-engine/pkg/util"
 )
 
+const defaultSectorSize int64 = 4096
```

Comment:
nit: many `defaultSectorSize` longhorn-engine codes, we can reorganize them in the future.

Reply:
Ticket created https://github.com/longhorn/longhorn/issues/4599. Thanks